modes/segmentation/src/index.tsx

```diff
@@ -132,7 +132,9 @@ function modeFactory({ modeConfiguration }) {
             id: ohif.layout,
```

Comment:
I think this is all that is needed to enable it in any mode, you can set it in the layout template @wayfarer3130

Reply:
I just tested it in ohif.thumbnailList and yes, just using that now works.